examples/resources/pingfederate_oauth_ciba_server_policy_settings/import.sh

```diff
@@ -0,0 +1,2 @@
+# "oauthCibaServerPolicySettingsId" should be "id"
```

Comment:
This comment is weird - is it saying it needs to be the literal string "id"?

Reply:
This is how it was on the other singular resources. I've updated all of their descriptions.